client/components/grades-form.jsx

```diff
@@ -1,83 +1,128 @@
-/* eslint-disable indent */
 import React from 'react';
 
 export default class GradeForm extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            name: '',
-            course: '',
-            grade: ''
-        };
-        this.handleChange = this.handleChange.bind(this);
-        this.handleSubmit = this.handleSubmit.bind(this);
-        this.handleReset = this.handleReset.bind(this);
-    }
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      course: '',
+      grade: ''
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleReset = this.handleReset.bind(this);
+  }
 
-    handleChange(event) {
-        const property = event.target.placeholder.toLowerCase();
-        this.setState({
-            [property]: event.target.value
-        });
-    }
+  handleChange(event) {
+    const property = event.target.placeholder.toLowerCase();
+    this.setState({
+      [property]: event.target.value
+    });
```

Comment:
Please also change this from `placeholder` to `name`. That way you can keep the casing in the placeholder, but don't have to mess with uppercasing or lowercasing if you format your `name` props as lowercase.